src/mongocxx/include/mongocxx/v1/return_document.hpp

```diff
@@ -0,0 +1,42 @@
+// Copyright 2009-present MongoDB, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include <mongocxx/v1/return_document-fwd.hpp>
+
+//
+
+#include <mongocxx/v1/detail/prelude.hpp>
+
+namespace mongocxx {
+namespace v1 {
+
+///
+/// `returnDocument` from the CRUD API specification.
+///
+/// @see
+/// - [CRUD API (MongoDB Specifications)](https://www.mongodb.com/docs/manual/reference/command/return_document/)
+///
+enum class return_document {};
```

Comment:
This replaces `v_noabi::options::find_one_common_options` as a more specific and direct component+class name describing its actual purpose (representing the `returnDocument` field option).